src/auth/index.ts

```diff
@@ -96,7 +96,7 @@ export default function createShopifyAuth(options: OAuthStartOptions) {
       try {
         await Shopify.Auth.validateAuthCallback(ctx.req, ctx.res, ctx.query);
 
-        ctx.state.shopify = await Shopify.Utils.loadCurrentSession(ctx.req, ctx.res);
+        ctx.state.shopify = await Shopify.Utils.loadCurrentSession(ctx.req, ctx.res, config.accessMode === 'online');
```

Comment:
Is there any benefit to changing the `AccessMode` type to an `enum`?  Being able write this as
```suggestion
        ctx.state.shopify = await Shopify.Utils.loadCurrentSession(ctx.req, ctx.res, config.accessMode === AccessMode.Online);
```
seems more maintenance friendly - using `online` and `offline` strings as checks seems like an opportunity to create errors in the future.

One step further - is there perhaps an opportunity to create an `isOnline(accessMode: AccessMode)` utility method?

Reply:
The enum looks nicer, but I don't know how likely it is that new modes will be added in the future. All of these suggestions make sense, but should we consider them in a separate issue / PR?

Reply:
I created https://github.com/Shopify/koa-shopify-auth/issues/60 since that change is slightly bigger than it seems at first.